### DISCUSSION TITLE
DOPS .itp file??

### DISCUSSION THREAD

**Discussion by jduro99** [Martini-Force-Field-Initiative/MartiniDiscussions#22] [Lipids]
I can't download the .itp of the DOPS lipid.

**Comment by ricalessandri** [comment]
https://github.com/marrink-lab/martini-forcefields/blob/784591ebdc91d762ed4df986c4650546c938f776/martini_forcefields/regular/v3.0.0/gmx_files/martini_v3.0.0_phospholipids_v1.itp#L3171
